freetype.nimble

```diff
@@ -1,9 +1,10 @@
-[Package]
-name          = "freetype"
-version       = "0.1.1"
+# Package
+version       = "0.2.0"
 author        = "Andri Lim"
 description   = "FreeType2 library wrapper"
 license       = "MIT"
+srcDir        = "src"
```

Comment:
please keep the directory structure as is. I don't like `src/freetype`, a simple `freetype` is ok.